data-products-dp0-2/index.rst

```diff
@@ -98,112 +95,68 @@ Coadd images are divided into ``tracts`` (a spherical convex polygon) and tracts
 
-**Schema:**
-A table's "schema" refers to the column names, units, and descriptions of the tabulated data.
-Links to full or curated versions of the table schema (curated meaning limited to columns that will be of most use to most DP0 delegates) are provided in the tables below.
-A `DP0.2 schema browser <https://dm.lsst.org/sdm_schemas/browser/dp01.html>`_ is also available, thanks to the Rubin Data Management team.
+**DP0.2 Table Schema:**
+The column names, units, and descriptions of the DP0.2 catalogs listed in the table below are all available via the `DP0.2 schema browser <https://dm.lsst.org/sdm_schemas/browser/dp02.html>`_.
+
+.. list-table:: Catalog data available for DP0.2.
+   :widths: 100 100 390
+   :header-rows: 1
+
+   * - TAP Name
+     - Butler Name
+     - Description
+   * - Object
+     - objectTable
+     - Astrometric and photometric measurements for objects detected in coadded images (990 columns).
+   * - Source
+     - sourceTable
+     - Astrometric and photometric measurements for sources detected in the individual PVIs (143 columns).
+   * - ForcedSource
+     - forcedSourceTable
+     - Forced photometry on the individual PVIs at the locations of all detected objects and sources (38 columns).
+   * - DiaObject
+     - diaObjectTable_tract
+     - Derived summary parameters for DiaSources associated by sky location, including lighcurve statistics (137 columns).
+   * - DiaSource
+     - diaSourceTable_tract
+     - Astrometric and photometric measurements for sources detected in the difference images (66 columns).
+   * - ForcedSourceOnDiaObject
+     - forcedSourceOnDiaObjectTable_tract
+     - Forced photometry on the individual PVIs at the locations of all DiaObjects (35 columns).
+   * - Visit
+     - visitTable
+     - Individual visit information, including band, airmass, exposure time, and so on (15 columns).
+   * - CcdVisit
+     - ccdVisitTable
+     - Individual CCD (detecor) information, including measured seeing, sky background, and zeropoint (30 columns).
+
+|
 
 **Principal Columns:**
 For convenience, Rubin Observatory staff have identified the principal columns which are most likely to be useful.
 These principal columns will be pre-selected in the Table View of the RSP's Portal Aspect.
-The table schema below also identify the principal columns.
 
 **Recommended Search Parameter "detect_isPrimary = True":**
-A good default search query parameter for catalogs of objects in the coadded images is to set **detect_isPrimary** = **True**, i.e., for the dp01_dc2_catalogs.position (TAP) and deepCoadd_ref (Butler) catalogs.
+A good default search query parameter for the Object, Source, and ForcedSource catalogs is to set **detect_isPrimary** = **True**.
 The ``detect_isPrimary`` parameter is ``True`` if a source has no children, is in the inner region of a coadd patch, is in the inner region of a coadd tract, and is not “detected” in a pseudo-filter.
```

Comment:
See [this link](https://pipelines.lsst.io/getting-started/multiband-analysis.html#filtering-for-unique-deblended-sources-with-the-detect-isprimary-flag) for more info?

Reply:
added